### PR TITLE
feat(landing page): add jumbotron

### PR DESCRIPTION
**Pull request by mnizarzr**
add jumbotron

### CODE REVIEW

**Review by alamsyhhh — CHANGES_REQUESTED** [review]

File: index.html [hunk 1]
```diff
@@ -71,6 +71,15 @@
       </div>
     </nav>
 
+  <!-- JUMBOTRON SECTION -->
+  <div class="container jumbotron py-5">
+    <h1 class="display-4">Welcome to our website!</h1>
+    <p class="lead">We provide amazing services for all your needs.</p>
```
Comment (fadhilahfarhan) [inline index.html:77]: text color change to red

File: index.html [hunk 2]
```diff
@@ -71,6 +71,15 @@
...
+    <hr class="my-4">
```
[the remark this answers is not in the document]
Comment (mnizarzr) [reply index.html]: sudah

[the remark this answers is not in the document]
Comment (alamsyhhh) [reply index.html]: done

File: index.html (hunk 1 above)
Comment (alamsyhhh) [reply index.html]: done

**Pull request merged**